extensions/sql-migration/src/wizard/assessmentDetailsPage/assessmentDetailsHeader.ts

```diff
@@ -158,8 +164,81 @@ export class AssessmentDetailsHeader {
 		}).component();
 
 		cardContainer.addItems([cardHeading, cardText]);
-		this._valueContainers.push(cardText);
 
+		if (linkMetaData.title === constants.RECOMMENDED_CONFIGURATION) {
+			this._linksContainer = this._view.modelBuilder.flexContainer().withProps({
+				CSSStyles: {
+					'display': 'flex',
+					'flex-direction': 'row',
+					'width': '170px'
+				}
+			}).component();
+
+
+			this._viewDetailsLink = this._view.modelBuilder.hyperlink().withProps({
+				label: constants.VIEW_DETAILS,
+				url: '',
+				height: 18,
+				CSSStyles: {
+					'font-size': '13px',
+					'font-weight': '400',
+					'line-height': '18px',
+					'text-decoration': 'none',
+					'display': 'none',
+				}
+			}).component();
+
+
+			this._viewDetailsLink.onDidClick(async () => {
+				if (hasRecommendations(this.migrationStateModel)) {
+					const skuRecommendationResultsDialog = new SkuRecommendationResultsDialog(this.migrationStateModel, this.migrationStateModel._targetType);
+					await skuRecommendationResultsDialog.openDialog(
+						this.migrationStateModel._targetType,
+						this.migrationStateModel._skuRecommendationResults.recommendations
+					);
+				}
+			});
+
+			this._saveTemplateLink = this._view.modelBuilder.hyperlink().withProps({
+				label: constants.TARGET_PROVISIONING_TITLE,
+				url: '',
+				height: 18,
+				CSSStyles: {
+					'font-size': '13px',
+					'font-weight': '400',
+					'line-height': '18px',
+					'text-decoration': 'none',
+					'display': 'none',
+				}
+			}).component();
+
+			this._saveTemplateLink.onDidClick(async () => {
+				if (hasRecommendations(this.migrationStateModel)) {
+					const generateProvisioningScriptDialog = new GenerateProvisioningScriptDialog(this.migrationStateModel, this.migrationStateModel._targetType);
+					await generateProvisioningScriptDialog.openDialog(/*
+						this.migrationTargetType,
+						this.migrationStateModel._skuRecommendationResults.recommendations*/
+					);
+				}
+			});
+
+			this._separator = this._view.modelBuilder.text().withProps({
```

Comment:
instead of using this separator better way is to use a toolbar component and put both buttons there. Toolbar has this separator already there. No need to make the change. You can remember this in future.